download/tests/support/mod.rs

```diff
@@ -31,26 +33,39 @@ pub fn write_file(path: &Path, contents: &str) {
     file.sync_data().expect("writing test data");
 }
 
+// This is a dead simple hyper server implementation, if we want to know more about
+// how to create a hyper server, we can check the following link:
+//
+// https://hyper.rs/guides/1/server/hello-world/
```

Comment:
This could be made even simpler :)

```suggestion
// A dead simple hyper server implementation.
// For more info, see:
// https://hyper.rs/guides/1/server/hello-world/
```